design-documents/frontend/technical-vision.md

```diff
@@ -181,9 +194,14 @@ _Disclaimer: the above list is not exhaustive and is in no particular order._
 
 Ideally, for each front-end project, there will be a separate project that covers both performance and critical-path testing. This project is separate so that it doesn't cripple the short feedback loops of feature development. Additionally, we would do well to have additional performance testing around the most commonly used components (e.g., used in 3+ places).
 
+## Scaffolding
+
+[Yeoman][] is the de facto standard for scaffolding modern web applications. In an effort to make the above guide as painless as possible and to encourage best practices, a [TSX Yeoman Generator](https://www.npmjs.com/package/generator-tsx) has been created for you. Refer to its documentation for more information.
```

Comment:
Would prefer if we remove `is the de facto standard`. I won't be too bike-sheddy here, but that's a strong endorsement of a tool that I don't see the FE community adopting much in recent years. We should probably just remain neutral here.

`Yeoman is a tool for scaffolding applications`.

Reply:
np